layouts/default.vue

```diff
@@ -17,7 +17,7 @@ const { currentTrack } = inject(MediaPlaylistInjectionKey)!;
     class="flex h-full max-h-[100vh] overflow-y-hidden bg-background-1 text-black-1 dark:bg-background-dark-1 dark:text-white-1"
   >
     <SidebarElement />
-    <main class="relative flex-grow overflow-y-auto">
+    <main class="relative flex-grow overflow-y-auto" style="overflow-x: hidden">
```

Comment:
Can we use a tailwind class here instead of inline styles?:)
See the `overflow-y-auto` class already present

Reply:
I'd avoid an overflow of `x`, if possible. Often this makes things like overlays or similar effects just more complicated.